frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortletUtils.java

```diff
@@ -107,12 +108,53 @@ public static void updateProjectFromRequest(PortletRequest request, Project proj
                 case EXTERNAL_IDS:
                     project.setExternalIds(PortletUtils.getExternalIdMapFromRequest(request));
                     break;
+                case TODOS:
+                    String userId = UserCacheHolder.getUserFromRequest(request).getId();
+                    updateProjectTodosFromRequest(request.getParameterValues(field.toString()), userId, project);
+                    break;
                 default:
                     setFieldValue(request, project, field);
             }
         }
     }
 
+    private static void updateProjectTodosFromRequest(String[] ids, String userId, Project project) {
+        Set<String> idSet = ids != null ? Arrays.stream(ids).collect(Collectors.toSet()) : Collections.emptySet();
+        Set<ProjectTodo> currentTodods = project.getTodosSize() > 0 ? project.getTodos() : Collections.emptySet();
```

Comment:
Just a little type in todos. If you should push some changes anyway, you could fix this as well.

Reply:
oops will fix